core/src/subgraph/provider.rs

```diff
@@ -137,3 +163,48 @@ impl<L, S> EventProducer<SubgraphProviderEvent> for SubgraphProvider<L, S> {
             .map(|s| Box::new(s) as Box<Stream<Item = SubgraphProviderEvent, Error = ()> + Send>)
     }
 }
+
+fn attribute_indexing_operations(
```

Comment:
I'm open to suggestions for a better place to put this function, I was very close to putting it in /graph/src/data/subgraph/schema.rs, but that would have required importing `graph_graphql` into graph.

Reply:
A method on `Schema` seems better, the only real depedency on `graph_graphql` I see is `get_valid_value_type` which you can put somewhere in `graph::data::graphql`, the rest you can get directly from `graphql_parser`.

As a general note, I've wanted to move `graph_graphql::schema` into `graph` before, to be able to do more in `Schema` methods.